#!/usr/bin/env python
PACKAGE = "phy_exp4"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

debug_image = gen.enum([gen.const("Template", int_t, 0, "Template"),
                        gen.const("Roi", int_t, 1, "Roi"),
                        gen.const("Mask", int_t, 2, "Mask"),
                        gen.const("Morphology", int_t, 3, "Morphology"),
                        gen.const("Circles", int_t, 4, "Circles")],
                       "channel")
gen.add("debug_image", int_t, 0, "Debug image", 0, 0, 4,
        edit_method=debug_image)

exit(gen.generate(PACKAGE, "phy_exp4", "Detection"))
